Cargo.toml

```diff
@@ -78,6 +78,7 @@ io-lifetimes = { version = "1.0.10", default-features = false, features = ["clos
 serial_test = "0.6"
 memoffset = "0.8.0"
 flate2 = "1.0"
+cfg-if = "1.0.0"
```

Comment:
I don't see `cfg_if` in use in the code; can this dependency be removed?

Reply:
Oopsy daisy, fixed